content/docs/components-and-props.md

```diff
@@ -16,23 +16,23 @@ prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
 
-Components let you split the UI into independent, reusable pieces, and think about each piece in isolation. This page provides an introduction to the idea of components. You can find a [detailed component API reference here](/docs/react-component.html).
+元件使你可以將 UI 拆分成獨立且可復用的程式碼，並且專注於各別程式碼的思考。本章節旨在介紹元件的相關概念，你也可以在此參閱[詳細的 API 文件](/docs/react-component.html)。
```

Comment:
Hey @bugtender ! I know you will fix this later. Just a quick reminder, we might want to consider using `component` in English rather than `元件` to avoid having to translate pureComponents and higherComponents. We should discuss this soon!